From ddcdb93215f9a19f2782842923da21ff8d817ae5 Mon Sep 17 00:00:00 2001
From: Gaku Inami <gaku.inami.xw@bp.renesas.com>
Date: Thu, 2 Mar 2017 14:20:53 +0900
Subject: [PATCH 158/909] soc: renesas: rcar_ems_ctrl: Fix build error with
 disabling CONFIG_CPU_FREQ

commit 97a7666960ceab82f2aa39748a65a25e3831742d from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

The build error has occurred in rcar_ems_ctrl.c as below when
CONFIG_CPU_FREQ is disabled. This patch adds "ifdef CONFIG_CPU_FREQ"
in rcar_ems_ctrl.c.

drivers/soc/renesas/rcar_ems_ctrl.c:235:2: error: implicit
declaration of function 'cpufreq_update_policy'

Note: This is local patch for R-Car Gen3 BSP, not for upstream.

Signed-off-by: Gaku Inami <gaku.inami.xw@bp.renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/renesas/rcar_ems_ctrl.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/soc/renesas/rcar_ems_ctrl.c b/drivers/soc/renesas/rcar_ems_ctrl.c
index 8295365075c5..bc6ce606cd29 100644
--- a/drivers/soc/renesas/rcar_ems_ctrl.c
+++ b/drivers/soc/renesas/rcar_ems_ctrl.c
@@ -232,7 +232,9 @@ static int rcar_ems_thermal_notifier_call(struct notifier_block *nb,
 		return NOTIFY_DONE;
 	}
 
+#ifdef CONFIG_CPU_FREQ
 	cpufreq_update_policy(cpumask_any(&freq_scaled_cpus));
+#endif
 
 	return NOTIFY_OK;
 }
-- 
2.17.1

